core/lexicon/en/access.inc.php

```diff
@@ -81,7 +81,7 @@
 $_lang['resource_group_add'] = 'Add Resource Group';
 $_lang['resource_group_access_remove'] = 'Remove Resource from Group';
 $_lang['resource_group_access_remove_confirm'] = 'Are you sure you want to remove this resource from the resource group?';
-$_lang['resource_group_create'] = 'Create Resource Group';
+$_lang['resource_group_create'] = 'Add Group';
```

Comment:
It is better not to change the lexicons for 2.x, firstly, the meaning does not change much here, and secondly, lexicons will appear that will need to be translated again in Crowdin.

Reply:
Ok, will resist lex changes in future for 2.x ;-)